p2p_wallet/Scenes/Main/Wormhole/Claim/WormholeClaimViewModel.swift

```diff
@@ -137,37 +139,31 @@ class WormholeClaimViewModel: BaseViewModel, ObservableObject {
         }
     }
 
-//    private func logAlert(for account: EthereumAccount, error: Swift.Error) {
-//        let token: ClaimAlertLoggerErrorMessage.Token = .init(
-//            name: account.token.name,
-//            solanaMint: nil,
-//            ethMint: account.token.tokenPrimaryKey,
-//            claimAmount: bundle.state.value?.resultAmount.asCryptoAmount.amount.description.double?.description
-//        )
-//
-//        DefaultLogManager.shared.log(
-//            event: "Wormhole Claim iOS Alarm",
-//            logLevel: .alert,
-//            data: AlertLoggerError(
-//                title: "Wormhole Claim iOS Alarm",
-//                message:
-//                    ClaimAlertLoggerErrorMessage(
-//                        tokenToClaim: token,
-//                        userPubkey: accountStorage.account?.publicKey.base58EncodedString ?? "",
-//                        userEthPubkey: ethModel?.account.address ?? "",
-//                        platform: "iOS \(UIDevice.current.systemVersion)",
-//                        appVersion: AppInfo.appVersionDetail,
-//                        timestamp: "\(Int64(Date().timeIntervalSince1970 * 1000))",
-//                        simulationError: nil,
-//                        bridgeSeviceError: error.readableDescription,
-//                        feeRelayerError: nil,
-//                        blockchainError: nil
-//                    )
-//              )
-//        )
-//    }
-
-//    var ethModel: WormholeClaimEthereumModel? { model as? WormholeClaimEthereumModel }
+    private func logAlert(for account: EthereumAccount, error: Swift.Error) {
```

Comment:
Is it duplication? I see 2 places that is logging `ClaimAlertLoggerErrorMessage`

Reply:
Not really, it is used to catch general error on WormholeClaim screen

Reply:
Oh. I see. Thanks